x-pack/plugins/security_solution_serverless/server/cloud_security/cloud_security_metering_task.ts

```diff
@@ -212,21 +219,20 @@ export const getCloudSecurityUsageRecord = async ({
   logger,
 }: CloudSecurityMeteringCallbackInput): Promise<UsageRecord[] | undefined> => {
   try {
```

Comment:
Following our decision to [prevent overbilling](https://github.com/elastic/security-team/issues/9560) and until we implement a [recovery](https://github.com/elastic/security-team/issues/9424) mechanism, `getSearchStartDate` is redundant.